src/App/Entity/Talk.php

```diff
@@ -77,57 +77,58 @@ public static function fromStandardTalk(
         DateTimeImmutable $time,
         Speaker $speaker,
         string $title,
-        string $abstract
+        string $abstract,
+        string $youtubeId = null
     ) : self {
         $talk = new self();
         $talk->meetup = $meetup;
         $talk->time = $time;
         $talk->speaker = $speaker;
         $talk->title = $title;
         $talk->abstract = $abstract;
+        $talk->youtubeId = $youtubeId;
         if ($talk->abstract === '') {
             $talk->abstract = null;
         }
+        if ($talk->youtubeId === '') {
+            $talk->youtubeId = null;
+        }
         return $talk;
     }
 
     /**
      * @param DateTimeImmutable $time
-     * @param Speaker $speaker
      * @param string $title
      * @param string $abstract
-     * @return void
```

Comment:
Yes, phpstorm stripped it :( it's not redundant, it's explicitly saying "this function returns nothing", and is only a docblock because we support PHP 7.0 still...when we go 7.1-only, we'll use ` : void` and this docblock can go ;)